client/styles/ratingsStyle.css

```diff
@@ -3,7 +3,7 @@
   flex-direction: row;
   font-size: 16px;
   font-weight: 300;
-  padding-top: 16px;
+  padding-top: 17px;
   /* padding-bottom: 10px; */
```

Comment:
Do you need these  CSS comments? Maybe consider removing if not needed.

Reply:
I've removed all comments from this css file, I'll comb through the others to remove comment's I don't want anymore as well. Thank you!